Automattic/wp-calypso
# Theme Showcase: Update WordPress.com Experts nudge to use UpsellNudge

#### Changes proposed in this Pull Request

* Following a long and tedious journey from #54140, #54522, and #54608, this updates the WordPress.com Experts nudge to use the UpsellNudge component

**Before**

<img width="1367" alt="Screen Shot 2021-07-13 at 1 32 53 PM" src="https://user-images.githubusercontent.com/2124984/125498902-673cf024-460b-4097-a4b0-f5bf590c679b.png">

**After**

Simple/Atomic

<img width="1363" alt="Screen Shot 2021-07-13 at 1 32 17 PM" src="https://user-images.githubusercontent.com/2124984/125498961-368aafa9-3de4-41ae-969e-be4ba89fda03.png">

Jetpack

<img width="1370" alt="Screen Shot 2021-07-13 at 1 32 36 PM" src="https://user-images.githubusercontent.com/2124984/125498965-248463b1-86a1-40c3-a54b-70ab8fca6059.png">

#### Testing instructions

* Switch to this PR, navigate to `/themes`
* Make sure clicking on the banner/button takes you to the "Built by WordPress.com" site: https://wordpress.com/built-by-wordpress-com/
* Make sure Tracks events fire as expected (impression, click, dismiss)

Related to #54087
